pxl_scripts/px/redis_flow_graph/redis_flow_graph.pxl

```diff
@@ -15,52 +16,131 @@ window_ns = px.DurationNanos(10 * ns_per_s)
 
 def redis_flow_graph(start: str, ns: px.Namespace, from_entity_filter: str, to_entity_filter: str):
 
+    # redis_events.beta is a beta table, which is subject to change.
     df = px.DataFrame('redis_events.beta', start_time=start)
-    df.pod = df.ctx['pod']
-    df.namespace = df.ctx['namespace']
-    df = df.drop(['upid'])
+    df = add_source_dest_columns(df)
 
     # Filter on namespace as specified by the user.
     df = df[df.namespace == ns]
 
+    # Filter out messages with empty source / destination.
```

Comment:
Does this happen in the data we have?

Reply:
I moved this logic to add_source_dest_columns(), so it'll be applied in all of the scripts. I haven't seen this issue on my Redis test clusters. According to Omid, this happens when the process traced was not a K8s pod.